src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -119,6 +119,54 @@ export const coreDeprecationProvider: ConfigDeprecationProvider = ({
   renameFromRoot('xpack.telemetry.config', 'telemetry.config'),
   renameFromRoot('xpack.telemetry.banner', 'telemetry.banner'),
   renameFromRoot('xpack.telemetry.url', 'telemetry.url'),
+  // Monitoring renames
```

Comment:
Can we add a note here about moving these to the monitoring plugin once it's been migrated?